lens.cabal

```diff
@@ -327,6 +327,7 @@ library
     Numeric.Natural.Lens
 
   other-modules:
+    Control.Lens.Internal.Prelude
```

Comment:
By convention, `lens` puts all its `Control.Lens.Internal` modules under `exposed-modules`. There's also another benefit in doing this: there is [some CPP in `benchmarks/plated.hs`](https://github.com/ekmett/lens/blob/eb255ec6c1e5c4f0012ab56268b43a3059d60a16/benchmarks/plated.hs#L10-L12) that could be done away with by importing `Control.Lens.Internal.Prelude` instead.

Reply:
I'd avoid that for a time being, until it's truly not changing that much. The `Control.Lens.Internal` module are way too stable

Reply:
The whole point of the `Internal` label is that we exempt it from PVP guarantees, right? Or an I missing something?

Reply:
I don't want to increase API surface of `lens`. I'd then rather remove the commit and span the CPP all over the place again.

Reply:
OK, but please make a note of the fact that `C.L.Internal.Prelude`

1. Is not exported, and
2. This convention differs from other `C.L.Internal` modules

In the Haddocks for `C.L.Internal.Prelude`.

Reply:
Done, also

```
    Depend on base-compat in plated benchmarks. criterion depends
    on base-compat-batteries anyway.
```